sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -341,16 +341,17 @@ private Combine() {
     /**
      * Adds the given input value to the given accumulator, returning the new accumulator value.
      *
-     * <p>For efficiency, the input accumulator may be modified and returned.
+     * @param accumulator may be modified and returned, for efficiency
```

Comment:
It would make this even more clear to rename the parameter to `mutableAccumulator`. Just a thought.